test/run

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
+# This is the expected entry point for Cockpit CI; will be called without
+# arguments but with an appropriate $TEST_OS, and optionally $TEST_SCENARIO
+
+# Prepare image
+if [[ $TEST_SCENARIO == container* ]]; then
```

Comment:
This is a bashism. Either use `#!/bin/bash` or use POSIX shell (`[ "${TEST_SCENARIO#container}" !1 "$TEST_SCENARIO" ]`). But let'sjust move these into  the `case` loop where you already pattern-match the scenario. Then you have all necessary commands (prepare and run) in one place, which is also easier to follow. Then `$TEST_SCENARIO` will also always have a value, so that `set -u` will work.